packages/plugin-ua-parser-browser/README.md

```diff
@@ -0,0 +1,6 @@
+<p align="center">
+  <a href="https://amplitude.com" target="_blank" align="center">
+    <img src="https://static.amplitude.com/lightning/46c85bfd91905de8047f1ee65c7c93d6fa9ee6ea/static/media/amplitude-logo-with-text.4fb9e463.svg" width="280">
+  </a>
+  <br />
```

Comment:
Can you please add a short description of the plugin here and link to the docs calling out the difference in 2.0?

Reply:
Hmmm, seems the UA parser change is not currently mentioned in the 2.0 migration docs. https://www.docs.developers.amplitude.com/data/sdks/browser-2/migration/